fly.toml

```diff
@@ -0,0 +1,67 @@
+app = "lucky-jumpstart"
+
+kill_signal = "SIGINT"
+kill_timeout = 5
+processes = []
+
+[build]
+  dockerfile = "dockerfiles/fly.Dockerfile"
+
+[deploy]
+  release_command = "lucky db.migrate"
+
+[env]
+  LUCKY_ENV = "production"
+
+[experimental]
+  allowed_public_ports = []
+  auto_rollback = true
+
+[[services]]
+  internal_port = 8080
+  processes = ["app"]
+  protocol = "tcp"
+  script_checks = []
+
+  [services.concurrency]
+    hard_limit = 50
+    soft_limit = 40
+    type = "connections"
+
+  [[services.http_checks]]
+    grace_period = "5s"
+    interval = 10000
+    method = "get"
+    path = "/ping"
+    protocol = "http"
+    timeout = 2000
+    tls_skip_verify = false
+
+    [services.http_checks.headers]
+      X-Forwarded-Proto = "https"
+
+  [[services.ports]]
+    handlers = ["http"]
+    port = 80
+
+  [[services.ports]]
+    handlers = ["tls", "http"]
+    port = 443
+
+  [[services.tcp_checks]]
+    grace_period = "1s"
+    interval = "15s"
+    restart_limit = 0
+    timeout = "2s"
+
+[[statics]]
```

Comment:
These three entries could be reduced to:

```
guest_path = "/app/public"
url_prefix = "/"
```

Reply:
Cool! I thought so, but I'm a bit unfamiliar with fly.io so I didn't know if this would break my API calls/SSR